### ISSUE TITLE
Healthy habits add up to 10 disease-free years to your life

### ISSUE THREAD

**Issue by nelsonic** [dwyl/health#124]
https://www.nhs.uk/news/lifestyle-and-exercise/healthy-habits-add-10-disease-free-years-your-life-study-reveals/l

+ not smoking
+ having a healthy body mass index (BMI) of 18.5 to 24.9
+ doing 30 minutes moderate to vigorous exercise a day
+ drinking alcohol only in moderation (defined in this study as just under 2 units for women and 4 units for men daily)
+ having a healthy diet score in the top 40% of people in the study

**Comment by iteles** [comment]
I would suggest sleep is missing from this list but fair 👍

**Comment by nelsonic** [comment]
@iteles agreed. Sleep is by _far_ the most important. But sleep data was not collected in the studies referenced in the article.
